extract-mislabeled-samples/extract_mislabeled_samples.py

```diff
@@ -0,0 +1,59 @@
+import re
+import pandas as pd
+from functools import reduce
+
+pd.set_option("display.max_rows", 1000)
+pd.set_option("display.max_colwidth", None)
+
+from datasets import load_dataset
+
+import os
+from pathlib import Path
+
+PROJECT_DIR = Path(os.path.abspath(__file__)).parent.parent
+
+
+def load_lang_predict(dataset_name: str, model_name: str, split: str):
+    # TODO: This structure/names of the lang prediction files might change
+    # TODO: Use the file with the ensembled lid instead of that of an individual model
+    lang_pred_filename = str(
+        Path(
+            PROJECT_DIR,
+            "predictions",
+            f"{re.sub('/', '_', dataset_name)}_{split}_{model_name}_predictions.csv",
+        )
+    )
+    lang_pred = pd.read_csv(lang_pred_filename)
+
+    for column in ["inputs_lid", "targets_lid"]:
+        lang_pred[column] = lang_pred[column].apply(lambda s: s[1:-1].split(":")[0])
+
+    return lang_pred
+
+
+if __name__ == "__main__":
+    dataset_name = "CohereForAI/aya_dataset"
```

Comment:
@AMR-KELEG @mecsabb I guess it would be great to parametrize these so that it could be provided via CLI and not hardcoded, similarly to how lid.py is set up: https://github.com/AMR-KELEG/Aya-mislabeled-samples/blob/master/lid.py#L28